versioning/scripts/cloudbuild/main.go

```diff
@@ -214,10 +240,20 @@ func newCloudBuildTemplateData(
 				break
 			}
 		}
-		data.AllImages = append(data.AllImages, images...)
+		// Ignore builder images from images list
+		if !v.Builder {
+			data.AllImages = append(data.AllImages, images...)
+		}
 		versionSTests, versionFTests := filterTests(structureTests, functionalTests, v)
-		data.ImageBuilds = append(
-			data.ImageBuilds, imageBuildTemplateData{v.Dir, images[0], images[1:], versionSTests, versionFTests})
+		// Enforce to use ImageNameFromBuilder as reference to create tags
```

Comment:
What is the purpose of ImageNameFromBuilder variable?

Reply:
It is used. It is need because image builder can create image with different name. this image name used as a reference to create new tags.

Reply:
Consumers of `imageBuildTemplateData` don't appear to use that field though, so perhaps it should be removed there and only kept in the `Version` struct.

Reply:
They use it. This field used as original tag which needed to create other tags for image.